user_modules/hmi_values.lua

```diff
@@ -322,7 +322,8 @@ function module.getDefaultHMITable()
         module.createButtonCapability("SEEKLEFT"),
         module.createButtonCapability("SEEKRIGHT"),
         module.createButtonCapability("TUNEUP"),
-        module.createButtonCapability("TUNEDOWN")
+        module.createButtonCapability("TUNEDOWN"),
+        module.createButtonCapability("SEARCH")
       },
       presetBankCapabilities = { onScreenPresetsAvailable = true },
     },
```

Comment:
Add in HMI values auto reply for buttonSubscription, for now SDL does not receive response by CUSTOM_BUTTON subscription during app registration